doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -11,44 +11,96 @@ menu:
     weight: 5
 ---
 
-Change Data Capture (CDC) allows you to track and propagate changes in a Postgres database to downstream consumers based on its Write-Ahead Log (WAL).
+This page shows you how to stream data directly from a PostgreSQL database to Materialize using PostgreSQL's [logical replication](https://www.postgresql.org/docs/10/logical-replication.html) protocol.
 
-This guide shows you how to propagate CDC data directly from [Postgres](/sql/create-source/postgres) to Materialize using Postgres’ native replication protocol.
+## Before you begin
```

Comment:
I think we could also use a section explaining the structure of how PG sources work in Materialize; even a quick diagram that shows:

```
                          tbl_a
                          /
PG -> MZ -> Primary source -> tbl_b
                          \
                          tbl_c
```

The reasoning behind this is that I think we ought to give users some context around what a "subsource" is and what it means; when they create the PG source we have them type "FOR TABLES," but then when they start working within MZ on these objects, the errors they might receive refer to these objects as subsources.

Would be glad to help you work with someone in design to get something drafted.